src/main/java/com/employee/app/view/ConsoleInput.java

```diff
@@ -0,0 +1,57 @@
+package com.employee.app.view;
+
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import com.employee.app.utils.exception.InvalidInputException;
+
+public class ConsoleInput implements InputView {
+	private static final Scanner SCANNER = new Scanner(System.in);
+	private static final String PHONE_NUMBER_REGEX = "^[0-9]{2,3}-[0-9]{3,4}-[0-9]{4}$";
+	private static final String EMAIL_REGEX = "(^[0-9a-zA-Z_-]+@[0-9a-zA-Z]+[.][a-zA-Z]{2,4}$)";
+
+	@Override
+	public String inputName() {
+		System.out.println("이름을 입력하세요:");
+		return SCANNER.nextLine();
+	}
+
+	private String validateInputInformation(String regex, String value) {
+		Matcher matcher = Pattern.compile(regex).matcher(value);
+		if(matcher.find())
+			return matcher.group();
```

Comment:
```suggestion
		if (matcher.find()) {
		    return matcher.group();
		}
```
저는 if를 사용하면 Curlybrace 를 항상 사용합니다.

Reply:
if를 사용할 때 중괄호를 열면 indent에 따라 버그가 발생할 수 있어서 닫으시는건가요? 가독성 측면에서 닫지 않고 여는게 좋다고 들어서 저는 열어놨었습니다만..

Reply:
이부분은 개인의 취향으로 생각할 수 있습니다. 다만 저는 사고를 예방하는 차원에서 괄호는 열고닫습니다.